src/structs.h

```diff
@@ -2372,6 +2365,8 @@ struct file_buffer
 #ifdef FEAT_DIFF
   int b_diff_failed; // internal diff failed for this buffer
 #endif
+
+  char_u *b_oni_line_comment;
```

Comment:
Not sure how OK it is to add a new field to the `buf_T` structure. But since it's at the end it should at least have relatively low chance of impacting FFI.

Reply:
Thanks for calling this out. Should be safe, as we're the only consumer of the library